client/my-sites/stats/post-performance/index.jsx

```diff
@@ -105,7 +105,7 @@ class StatsPostPerformance extends Component {
 				{ siteId && post && (
 					<QueryPostStats siteId={ siteId } postId={ post.ID } fields={ [ 'views' ] } />
 				) }
-				<SectionHeader label={ translate( 'Latest Post Summary' ) } href={ summaryUrl } />
+				<SectionHeader label={ translate( 'Latest post summary' ) } href={ summaryUrl } />
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.